superset/assets/spec/helpers/shim.js

```diff
@@ -17,7 +17,8 @@
  * under the License.
  */
 /* eslint no-native-reassign: 0 */
-import '@babel/polyfill';
```

Comment:
This got deprecated in favor of the new approach described here https://www.npmjs.com/package/@babel/polyfill